docs/admin/addons.rst

```diff
@@ -433,12 +429,12 @@ Additionally, the following environment variables are available:
 Post update repository processing
 ---------------------------------
 
-Post update repository processing can be used to update translation files on
-the source change. To achieve this, please remember that Weblate only sees
-files which are committed to the VCS, so you need to commit changes as a part
+Post update repository processing can be used to update translation files when
+the source changes. To achieve this, please remember that Weblate only sees
```

Comment:
The source of?

Reply:
probably better to speak about "upstream repository" here